spec/puppet-lint/lexer_spec.rb

```diff
@@ -18,7 +18,7 @@
     end
 
     it 'should calculate the line number for a multi line string' do
-      @lexer.instance_variable_set('@line_no', 2)
+      token = @lexer.new_token(:SSTRING, "test\ntest", 4)
```

Comment:
Doesn't actually affect the test, but just for correctness can you change the 3rd argument passed to `new_token` here from `4` to `9`?